tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "outDir": "dist",
     "rootDir": "src",
     "target": "es2017",
-    "strict": true
+    "strict": true,
+    "lib": ["es2021"]
   },
```

Comment:
I see you added the `lib` option in `tsconfig.json`. Why make this change and which features from ES2021 you want to use?

Reply:
it enables us to use `replaceAll` function